### ISSUE TITLE
Shadowfang keep looting issue

### ISSUE THREAD

**Issue by idkm23** [dalaranwow/dalaran-wow#5409]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 
Where: Shadow Fang Keep, in Archmage Arugal room
When: After killing Archmage Arugal, 4/17/2018 11:00 PM EST
What: 3 members attempted to loot the Archmage Arugal, and could not. The corpse was sparkling. Instead, they were put into a lingering crouching animation, with no looting GUI popup. However, one party member could loot it and we rolled for need/greed and that went fine. 
Why: Unknown
Who: Me and two other confirmed party members, I'm not sure about the other party members. The party was formed using the dungeon finder.

**Expected behaviour**: Tell us what should happen instead
I should be able to right click the Archmage Arugal corpse, and his inventory should show. If the corpse is empty, then I shouldn't get a weird permanent looting animation.

**Steps to reproduce the problem**:

1.  Not sure, queue Shadow Fang Keep?

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Can't reproduce. Make sure you have the quest in your log before the boss is killed.
Closing
